### DISCUSSION TITLE
Help i put all the token, seesion and server but it didn't work

### DISCUSSION THREAD

**Discussion by TakGamingYT** [sleeyax/aternos-discord-bot#57] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/94523932/183791361-4c732194-d59a-4777-b7a4-620388f41a90.png)

**Comment by sleeyax** [comment, marked as answer]
You put it wrong; os.getenv gets the environment variable so do don't enter your tokens in there! Code it like session = "value"

Also, replit is not supported for running the bot because it's IP banned by aternos/cloud flare, so it what you're trying to do won't work anyways.
